.cargo/config_fast_builds.toml

```diff
@@ -1,17 +1,68 @@
 # Copy this file to `config.toml` to speed up your builds.
 #
-# This configuration enables the following:
-#
-# - Faster linker (stable)
-#   - LLD is a faster drop-in replacement for the default linker, part of the LLVM project.
-#     It requires installation, please see the relevant section of this config for your platform.
-#   - Mold / Sold is a linker faster than even LLD, but with limited platform support.
-#     It requires installation, please see the relevant section of this config for your platform.
-#     Mold is precompiled for Linux, but must be manually built on MacOS and is unavailable for Windows.
-#   - Please note that LLD and Mold are incompatible with one another. Pick one and comment out the other!
-# - Nightly (unstable) features
-#   - `share-generics` prevents duplicating monomorphized generic code across crates.
-#   - `threads` makes use of the parallel frontend to increase compiler performance.
+# # Faster linker
+#
+# One of the slowest aspects of compiling large Rust programs is the linking time. This file configures an
+# alternate linker that may improve build times. When choosing a new linker, you have two options:
+#
+# ## LLD
+#
+# LLD is a linker from the LLVM project that supports Linux, Windows, MacOS, and WASM. It has the greatest
+# platform support and the easiest installation process. It is enabled by default in this file, so do not
+# worry about modifying anything!
+#
+# To install, please scroll to the corresponding table for your target (eg. `[target.x86_64-pc-windows-msvc]`
+# for Windows) and follow the steps under `LLD linker`.
+#
+# For more information, please see LLD's website at <https://lld.llvm.org>.
+#
+# ## Mold / Sold
+#
+# Mold is a newer linker written by one of the authors of LLD. It boasts even greater performance, specifically
+# through its high parallelism, though it only supports Linux and MacOS. On MacOS, you will need to download a
+# fork (maintained by the same author) named Sold and compile it yourself. (Note that Sold is for personal use
+# only, and has a paid commerical license.)
+#
+# Due to the limited platform support and adverse nature on MacOS, Mold disabled by default in this file. If you
```

Comment:
I would prefer comments to stay neutral on platforms

Reply:
Fixed in c8b9047000e88b35a8189921f33011ca52750a8f. I didn't intend on making it sound like MacOS is a worse platform (I even use it!), but I can see how it would be interpreted that way. Thanks for the catch!